src/main/java/uk/gov/justice/digital/delius/controller/secure/AppointmentBookingController.java

```diff
@@ -99,6 +101,25 @@ public AppointmentRescheduleResponse rescheduleAppointmentWithContextName(final
         return appointmentService.rescheduleAppointment(crn, appointmentId, context, appointmentRescheduleRequest);
     }
 
+    @RequestMapping(value = "/offenders/crn/{crn}/appointments/{appointmentId}/relocate",
+        method = RequestMethod.POST,
+        consumes = "application/json")
+    @ApiResponses(
+        value = {
+            @ApiResponse(code = 200, message = "Updated", response = String.class),
+            @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+            @ApiResponse(code = 403, message = "Requires role ROLE_COMMUNITY_INTERVENTIONS_UPDATE"),
```

Comment:
Perhaps this could have a less specific role in the future as it's not particularly interventions bespoke, and could maybe more widely used, unlike some of the other endpoints in this controller.

Reply:
@mxco86 do you have a feel for what other roles could be added to make this less specific to interventions?

Reply:
If and endpoint is more general it usually requires 'ROLE_COMMUNITY'

Reply:
I suggest leaving ROLE_COMMUNITY for read-only endpoints. Rightly or wrongly (wrongly IMHO) we are not using scope - e.g read,write - to control access to our endpoints, just role.

Given that maybe something like `ROLE_COMMUITY_APPOINTMENTS_UPDATE` so the update endpoints are at least locked to certain clients. For now, give everything else is `ROLE_COMMUNITY_INTERVENTIONS_UPDATE` maybe it doesn't matter until smoother client needs to use this endpoint.